### PR TITLE
Jest test jc

### PR DESCRIPTION
**Pull request by JAbreu96**

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -30,16 +31,27 @@ class App extends React.Component {
...
+    console.log(relatedStyle);
+    var status = !this.state.comparisonToggle ? <Comparison_Model toggleComparison={this.comparisonToggle} displayedProduct={this.state.currentProduct} relatedProduct={relatedProduct} /> : false;
 
+    this.setState({
+      comparisonToggle: status
+    })
+  }
 
   render() {
+    var comparison = this.state.comparisonToggle ? this.state.comparisonToggle : <div></div>;
```
Comment (mpxplore97) [inline client/components/App.jsx:45]: this is cool! never thought about using render like this

File: client/components/RelatedProdList/Comparison_Model.jsx [hunk 2]
```diff
@@ -1,16 +1,78 @@
...
+    }
+  }
+
+  console.log(Object.keys(displayInfo).length);
+  if (Object.keys(displayInfo).length) {
+    return (
+      <div className="comparison-viewer" onClick={(e) => {
+        props.toggleComparison();
+      }}>
+        <table className="comparison-table">
```
Comment (mpxplore97) [inline client/components/RelatedProdList/Comparison_Model.jsx:38]: great organization!

File: client/components/RelatedProdList/RelProductCard.jsx [hunk 3]
```diff
@@ -48,18 +48,26 @@ class RelProductCard extends React.Component {
       })
   }
   render() {
-    console.log(this.state.currentStyle);
```
Comment (mpxplore97) [inline client/components/RelatedProdList/RelProductCard.jsx:51]: Don't forget to take this out

File: server/index.js [hunk 4]
```diff
@@ -48,9 +48,21 @@ app.get('/products/:product_id/related', (req, res) => {
     .catch((err) => {
       console.log(err);
     })
+})
 
+app.get('/products/:product_id/info', (req, res) => {
+  console.log('ID', req.params);
```
Comment (mpxplore97) [inline server/index.js:54]: Don't forget to take out this console.log

File: client/components/RelatedProdList/Related_Outfit.css [hunk 5]
```diff
@@ -1,75 +1,192 @@
...
-  left: 90%;
+  left: 97%;
 }
 .related-list-arrows__left {
-  left: -2%;
+  left: 0%;
+}
+
+/* ARROW HOVER EFFECTS */
+.hvr-forward {
```
Comment (mpxplore97) [inline client/components/RelatedProdList/Related_Outfit.css:47]: Css Mastery! good job

**Review by mpxplore97 — CHANGES_REQUESTED** [review]
Awesome job. looks good!

File: client/components/App.jsx (hunk 1 above)
Comment (JAbreu96) [reply client/components/App.jsx:45]: Yeah its pretty useful!

File: client/components/RelatedProdList/RelProductCard.jsx (hunk 3 above)
Comment (JAbreu96) [reply client/components/RelatedProdList/RelProductCard.jsx:51]: Removed it in my current branch!

File: client/components/RelatedProdList/Related_Outfit.css (hunk 5 above)
Comment (JAbreu96) [reply client/components/RelatedProdList/Related_Outfit.css:47]: TY!

**Review by JAbreu96 — COMMENTED** [review]
Made changes to my current branch!

**Review by mpxplore97 — APPROVED** [review]
Looks good!

**Pull request merged**